extensions/bitwarden/src/search.tsx

```diff
@@ -71,23 +67,43 @@ function ItemList(props: {
     }
   }, [sessionToken]);
 
+  async function refreshItems() {
+    if (sessionToken) {
+      const toast = await showToast(ToastStyle.Animated, "Syncing Items...");
+      await bitwardenApi.sync(sessionToken);
+      await loadItems(sessionToken);
+      await toast.hide();
+    }
+  }
+
+  const favoriteItems = [];
+  const regularItems = [];
+  for (const item of state?.items || []) {
+    item.favorite ? favoriteItems.push(item) : regularItems.push(item);
+  }
```

Comment:
We don't need to `map` over both the favorites and regular items below. We can DRY things up and slightly improve the latency by doing it here:
```suggestion
  for (const item of state?.items || []) {
  	const listItem = <ItemListItem
  		key={item.id}
  		item={item}
  		folder={item.folderId ? folderMap[item.folderId] : undefined}
  		refreshItems={refreshItems}
  	/>
  		
    item.favorite ? favoriteItems.push(listItem) : regularItems.push(listItem);
  }
```